Components/Modal/modal.css

```diff
@@ -0,0 +1,50 @@
+.modal-container{
+    position: fixed;
+    display: none;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgb(0, 0, 0); /* Fallback color */
+    background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+    z-index: 100;
+}
+.modal-content{
+    width: 500px;
+    background-color: var(--background-color);
+    min-height: 300px;
+    border-radius: 15px;
+    margin: 0rem 1rem 0rem 1rem;
+}
+.modal-title{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 70px;
+    border-bottom: 1px solid black;
+}
+.modal-title h4{
+    margin-left: 10px;
+}
+.modal-title i{
+    font-size: 30px;
+    cursor: pointer;
+    margin-right: 10px;
+}
+.modal-text{
+    margin: 1rem 10px 1rem 10px;
```

Comment:
If the text is of 2-3 lines then it shows white space. You can reduce the min-height a little bit as mostly the text in the modal will be of 2-3 lines only.

Reply:
Other than modal-text everything is looks good to me

Reply:
Done with the changes..😁

Reply:
Looking great 🚀.